src/serializer.js

```diff
@@ -17,18 +19,19 @@ var Serializer = DS.RESTSerializer.extend({
   extractDeleteRecords: function(store, type, payload) {
     return this.extractArray(store, type, payload);
   },
-  serialize: function(snapshot, options) {
+  serialize: function(record, options) {
```

Comment:
TODO: do not create snapshot inside serialize method, betted do it create/update/...Record methods